### ISSUE TITLE
Sample Scene has multiple "The referenced script on this Behaviour [...] is missing!" warnings. (Broken)

### ISSUE THREAD

**Issue by seannowotny** [Hertzole/advanced-level-editor#18]
**Bug description:**
The sample scene has 86 of these warnings
As a result the sample scene doesn't work properly

**To reproduce:**
Import sample with package manager
Open "ALE Completed Sample" in Assets/Samples/Advanced Level Editor/0.1.0-preview/Completed Scene
Press Play
Observe warnings in console

**Expected behavior:**
No warnings
The sample Scene works as expected

**Screenshots:**
![Unity_lD7f5d1Of4](https://user-images.githubusercontent.com/21989324/226123253-93f4c7aa-504b-4428-a7b1-44ee9951c674.png)

**Desktop and versions:**
 - Your OS: Windows 10 Home 10.0.19044 Build 1944
 - Unity version: 2022.2.8

**Comment by seannowotny** [comment]
Exception in screenshot is unrelated to problem
![Unity_Y8Z2Lt0tV2](https://user-images.githubusercontent.com/21989324/226123362-615fb596-0818-4313-af40-8db2af163f88.png)

**Comment by seannowotny** [comment]
I was able to fix the exception by deleting the contents of AppData\LocalLow\[CompanyName]\[ProjectName]

However, the warnings remain and during play mode, this screen is what I'm left with. 
Clicking on the windows or trying to drag them around has no effect. 
![Unity_Kl327t9Op9](https://user-images.githubusercontent.com/21989324/226164999-bba00866-4b9f-4267-b475-f4c8b851b523.png)

The only exceptions are the checkmark, which can be toggled. I can also focus and type into the input field.
![Unity_q6pGVHpIGo](https://user-images.githubusercontent.com/21989324/226165100-8543648f-7424-4691-b8e6-3fcf15681270.png)

**Comment by seannowotny** [comment]
I was able to fix the missing references and missing fields by cross-referencing it with this repository opened with Unity 2020.33.3

**Comment by seannowotny** [comment]
Let me know if I should send you the fixed scene.
Perhaps the scene couldn't be loaded correctly because of the particular version of Unity I'm using for some reason 🤷‍♂️

**Comment by Hertzole** [comment]
Hi there!
I imported the package and the sample in 2022.2.10 and didn't get any of the issues you described. I did, however, notice that the scene doesn't seem to work correctly. There are some unknown problems going on. 
Unfortunately, I probably won't try to fix it either. I haven't worked on this in a long time and it's not in my interest to work on this currently. I'm sorry for the inconveniencies!
